### PR TITLE
Update readme.python.md

### PR DESCRIPTION
**Pull request by msyyc**
https://github.com/Azure/sdk-release-request/issues/3732

### CODE REVIEW

File: specification/storagemover/resource-manager/readme.python.md [hunk 1]
```diff
@@ -3,21 +3,16 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-``` yaml $(track2)
+``` yaml $(python)
```
Comment (changlong-liu) [inline specification/storagemover/resource-manager/readme.python.md:6]: Hi @msyyc , I'm wonderring shall we merge line 6~18 to same yaml block?

File: specification/storagemover/resource-manager/readme.python.md [hunk 2]
```diff
@@ -3,21 +3,16 @@
...
 package-name: azure-mgmt-storagemover
-no-namespace-folders: true
+namespace: azure.mgmt.storagemover
 package-version: 1.0.0b1
 clear-output-folder: true
 ```
 
-``` yaml $(python-mode) == 'update' && $(track2)
+``` yaml $(python)
 no-namespace-folders: true
```
Comment (changlong-liu) [inline specification/storagemover/resource-manager/readme.python.md:16]: Do we still need this line?

File: specification/storagemover/resource-manager/readme.python.md (hunk 1 above)
Comment (msyyc) [reply specification/storagemover/resource-manager/readme.python.md:6]: we want keep same with [template file](https://github.com/Azure/azure-rest-api-specs/tree/main/documentation/samplefiles) and existing python.md

File: specification/storagemover/resource-manager/readme.python.md (hunk 2 above)
Comment (msyyc) [reply specification/storagemover/resource-manager/readme.python.md:16]: Yes. It is related with autorest behavior.

**Review by Wzb123456789 — APPROVED** [review]

**Pull request merged**